.circleci/config.yml

```diff
@@ -48,6 +48,8 @@ jobs:
           command: |
             export PACKAGE_JSON_VERSION=$(node -e "console.log(require('./package.json').version)")
             export PUBLISH=$([[ "${CIRCLE_TAG:-}" == "v${PACKAGE_JSON_VERSION}" ]] && echo "On" || echo "Off")
+            echo ${CIRCLE_TAG:-}
+            echo ${PACKAGE_JSON_VERSION}
```

Comment:
Should we add `npm test` to these CI builds?

Reply:
This is addressed in https://github.com/valhalla/valhalla/pull/1415/files#diff-15547c54d3d4898a882b4ab7b3cee381R148, ignore.